README.md

```diff
@@ -92,7 +91,11 @@ All notable changes to this project will be documented in the [CHANGELOG.md](htt
 
 ## Authors
 
-See the [AUTHORS](https://github.com/lomsa-dev/http-mock-adapter/blob/master/AUTHORS "Project's AUTHORS file") file for information regarding the authors of the project.
+**NOTICE**: The file doesn't exist for now.
```

Comment:
I think it would be better to either remove the sections that do not have content or add them and them remove notices like these. We could file an issue for the _ACKNOWLEDGMENTS.md_ file.

Reply:
It was rebase issue

Reply:
Fixed